spec/integration/user_index_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+RSpec.describe 'User index integration tests', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+  before(:all) do
+    @users = User.all
+  end
+  it 'shows all the properties (Name, photo, number of posts) of each user' do
+    visit users_path
+    @users.each do |user|
+      expect(page).to have_text(user.Name)
+      expect(page).to have_content('Number of posts')
+
+      expect(page).to have_text(user.PostsCounter)
+    end
+  end
```

Comment:
- [ ] As the previous reviewer suggested that you are missing an test and test is if you click on any of the user then it takes you to the user show page :+1: Kindly add a test for that :blush: